rasa/shared/constants.py

```diff
@@ -94,3 +94,5 @@
 
 RESPONSE_CONDITION = "condition"
 CHANNEL = "channel"
+DEFAULT_YAML_SEQUENCE_INDENT_ENV_VAR = "DEFAULT_YAML_SEQUENCE_INDENT"
```

Comment:
I looked at the GH issue and Jira ticket but I couldn't find the request to make this configurable, isn't it better to enforce an indentation of 2 at all times instead? That seems to have been the observation in the GH issue too: `The important difference, which can be hardly seen here, are the two extra spaces required for list entries. ` 🤔

Reply:
Otherwise looks great 👍🏻

Reply:
The implementation in Rasa X made it configurable, so I followed the same pattern. The request wants an indentation of 4 (which, when the dash is added, looks like two), but the default in Rasa up till now was 2. We don't want to change the behaviour for all users, only the ones who use internal yaml formatters that require specific indentation.